src/systems/systems/src/hydraulic/linear_actuator.rs

```diff
@@ -2539,7 +2539,7 @@ mod tests {
     fn linear_actuator_not_moving_on_locked_rigid_body() {
         let mut test_bed = SimulationTestBed::new(|context| {
             let tested_object = cargo_door_assembly(context, true);
-            TestAircraft::new(context, Duration::from_millis(33), tested_object)
+            TestAircraft::new(context, Duration::from_millis(10), tested_object)
```

Comment:
I don't know if it would be helpful to extract these values to an "global" constant so it is changed later one only has to change it at one point.